wagtail/admin/checks.py

```diff
@@ -164,3 +164,22 @@ def check_panels_in_model(cls, context="model"):
         errors.append(error)
 
     return errors
+
+
+@register("wagtail_admin_base_url")
```

Comment:
should we create a future task to prefix all the other checks' names?

Reply:
I'm not intending this to set a precedent of all our checks starting with `wagtail_`, if that's what you mean - it just seemed logical to give it the same name as the setting it was checking for. (So really it should be `wagtailadmin_base_url` - I kept tripping up on the underscores while writing this, and I guess this one got left in...)